File: algo-daily/coding-solution/valid_palindrome.cpp
class Solution {
public:
    
    bool isPalindrome(string s) {
        
        int i = 0;
        int e = s.length()-1;
        
        while(i<e){
            s[i] = tolower(s[i]);
            s[e] = tolower(s[e]);
            if(!isalnum(s[i])){
                i++;
                continue;
            }
            if(!isalnum(s[e])){
                e--;
                continue;
            }
            if(s[i]!=s[e]){
                return false;
            }
            i++;
            e--;
        }
        
        return true;
        
    }
};

File: python-daily/valid_palindrome.py
class Solution:
    
    def isPalindrome(self, s: str) -> bool:
        temp = ""
        for x in s:
            if (x>='a' and x<='z') or(x>='A' and x<='Z') or (x>='0' and x<='9'):
                temp +=x
        temp = temp.lower()
        reverse_temp = temp[::-1]
        
        return reverse_temp==temp
        